mitodl/odl-video-service
# Multiangle VideoJS

#### What are the relevant tickets?
Closes #283

#### What's this PR do?
Enables multiangle playback with Video.js and removes the USwitch player.

#### How should this be manually tested?
- In the admin console, set a video to multiangle=True (ideally one that really is multiangle, but it's not necessary) - one is available with subtitles on [dropbox](https://www.dropbox.com/sh/1dk25zf882skcrc/AAAb8xCJad7nJgsGMirAvbTra?dl=0).
- Go to the video detail page for that video.  The video player should show one corner of the video, with 4 smaller screens to the right showing each angle of the video, with the one currently playing highlighted in red.
- Turn subtitles on. They should be visible/readable.
- Click on one of the other screens.  It should become highlighted in red and the main video should switch to that screen view.  Subtitles should still be on.
- Click the full screen icon.  Both the main video screen and 4 corner screens should be viewable in fullscreen mode, and clicking on each corner screen should still function the same.
- Either click the fullscreen icon again or press the Esc key.  The page should return to it's previous state.
- Resize the browser window.  The main video window and 4 corner screens should adapt in size.
- Try the above in different browsers.
- Load a video detail page for a video that is not multi-angle.  It should display normally (no corner screens, entire video width/height visible).
